cl_dll/hud.cpp

```diff
@@ -937,3 +937,12 @@ float CHud::GetSensitivity( void )
 	return m_flMouseSensitivity;
 }
 
+void CHud::ConvertToLowerCase(const char *str)
```

Comment:
This doesn't need to be a class method (let alone a non-static one). I think one place where I put auxiliary functions was https://github.com/YaLTeR/OpenAG/blob/master/cl_dll/cl_util.h, could be moved there.